tests/test_hear_cleanup.py

```
@@ -95,25 +140,32 @@ def test_cleanup_on_exit(self):
-				p.join(self.ERROR_JOIN_TIMEOUT)
+				p.join(self.KILL_DELAY_SECONDS)
 				if p.is_alive():
 					p.terminate()
 					p.close()
 				raise unittest.SkipTest(fail_fixture) from _cause
-			p.join(self.FINAL_JOIN_TIMEOUT)
+			p.join(self.PROCESS_TIMEOUT_SECONDS)
 			self.assertIsNotNone(p.exitcode)
-			self.assertEqual(int(p.exitcode), int(0))
-			theResult = (int(p.exitcode) <= int(0))
+			self.assertEqual(
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)